public/docs/_examples/gettingstarted/ts/package.json

```diff
@@ -14,7 +14,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "angular2": "2.0.0-alpha.39",
+    "angular2": "2.0.0-alpha.40",
```

Comment:
True but the other GS changes matter more.  And soon it will be 42 :-).  Please accept this as my next PR builds on it.